src/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+  </head>
+  <body>
+Hello, world
+  </body>
+</html>
```

Comment:
Don't forget to add a newline at the end
![image](https://user-images.githubusercontent.com/96409563/221644916-1c44686d-116b-4ca0-b4e8-d637cdd32631.png)
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file